src/org/jitsi/impl/neomedia/codec/video/vp8/DePacketizer.java

```diff
@@ -46,49 +46,11 @@
     private static final boolean TRACE = logger.isTraceEnabled();
 
     /**
-     * A <tt>Comparator</tt> implementation for RTP sequence numbers.
-     * Compares <tt>a</tt> and <tt>b</tt>, taking into account the wrap at 2^16.
-     *
-     * IMPORTANT: This is a valid <tt>Comparator</tt> implementation only if
-     * used for subsets of [0, 2^16) which don't span more than 2^15 elements.
-     *
-     * E.g. it works for: [0, 2^15-1] and ([50000, 2^16) u [0, 10000])
-     * Doesn't work for: [0, 2^15] and ([0, 2^15-1] u {2^16-1}) and [0, 2^16)
-     *
-     * NOTE: An identical implementation for Integers can be found in the class
-     * SeqNumComparator. Sequence numbers are 16 bits and unsigned, so an
-     * Integer should be sufficient to hold that.
-     */
-    private static final Comparator<? super Long> seqNumComparator
-            = new Comparator<Long>() {
-        @Override
-        public int compare(Long a, Long b)
-        {
-            if (a.equals(b))
-                return 0;
-            else if (a > b)
-            {
-                if (a - b < 32768)
-                    return 1;
-                else
-                    return -1;
-            }
-            else //a < b
-            {
-                if (b - a < 32768)
-                    return -1;
-                else
-                    return 1;
-            }
-        }
-    };
-
-    /**
      * Stores the RTP payloads (VP8 payload descriptor stripped) from RTP packets
      * belonging to a single VP8 compressed frame.
      */
-    private SortedMap<Long, Container> data
-            = new TreeMap<Long, Container>(seqNumComparator);
+    private SortedMap<Integer, Container> data
```

Comment:
does this need to be `Integer` (as opposed to `int`)?  thinking about all those Integer allocations from int conversions...